lotus/client_test.go

```diff
@@ -35,14 +35,6 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
```

Comment:
Deleted some unnecessary Lotus API mundane tests that aren't used in any part of the logic.
This file it's a bit paranoid in general since it's testing raw Lotus API calls... but trying to remove some to make complete pipeline testing a bit shorter.